src/webrtc/call.ts

```diff
@@ -1606,10 +1627,14 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
             this.pushLocalFeed(feed);
         }
 
+        this.preallocateTransceivers();
+
         if (requestScreenshareFeed) {
-            this.peerConn!.addTransceiver("video", {
-                direction: "recvonly",
-            });
+            for (const kind of ["audio", "video"]) {
+                this.peerConn!.addTransceiver(kind, {
+                    direction: "recvonly",
+                });
```

Comment:
Hmm interesting. We creating every time a new transceiver, when we receive a screenshare. I was thinking thats this will done automatically.. I did not know that